lib/get/util/clone.js

```diff
@@ -7,7 +7,7 @@ module.exports = function clone(node) {
     outValue = {};
     for (i = 0, len = keys.length; i < len; i++) {
         var k = keys[i];
-        if (k[0] === prefix) {
+        if (k[0] === prefix || k === "$absolutePath") {
```

Comment:
We need the prefix check here, because we're still using the prefix on cache meta-data currently? Just not on the JSON payload meta-data?